tests/test_lm/test_unit/test_base.py

```diff
@@ -58,3 +59,23 @@ def test_get_mpi_info(mocked_init):
     version, flavor = lm._get_mpi_info('mpirun')
     assert version == '2.1.2'
     assert flavor == 'OMPI'
+
+    ru.sh_callout.side_effect = [['test',1,1],['HYDRA build details:',3,0]]
+    version, flavor = lm._get_mpi_info('mpirun')
+    assert version == ''
+    assert flavor == 'HYDRA'
+
+    ru.sh_callout.side_effect = [['test',1,1],['Intel(R) MPI Library for Linux* OS,\n\n\
+                                               Version 2019 Update 5 Build 20190806\n\n\
+                                               Copyright 2003-2019, Intel Corporation.',3,0]]
+    version, flavor = lm._get_mpi_info('mpirun')
+    assert version == ''
+    assert flavor == 'HYDRA'
+
+    ru.sh_callout.side_effect = [['test',1,1],['MVAPICH2 2.3b',3,0]]
+    version, flavor = lm._get_mpi_info('mpirun')
+    try: 
```

Comment:
@AymenFJA which resource has this MPI version?

Reply:
XSEDE Bridges.